scripts/common.py

```diff
@@ -35,6 +35,7 @@
 from core import constants
 
 from typing import Dict, Generator, List, Optional, Union
+from scripts import servers  # isort:skip
```

Comment:
Why skip?

Reply:
I was copying code from a different file and I think this got copied as well. I have removed it.